website/sidebars.js

```diff
@@ -13,10 +13,31 @@ module.exports = {
       "advanced/user-object",
       "advanced/public-api",
       "advanced/caching",
-      "advanced/team-collaboration",
+      {
+        "User Management": [
+          "advanced/user-management/team-collaboration",
+          {
+            "SAML Single Sign-On": [
+              "advanced/user-management/saml/sso-overview",
```

Comment:
routes should reflect on the final naming of the feature

Reply:
✔️